scripts/ci/docker-test.sh

```diff
@@ -51,20 +51,26 @@ docker info
 
 criu --version
 
-# shellcheck disable=SC2016
-docker run --tmpfs /tmp --tmpfs /run --read-only --security-opt seccomp=unconfined --name cr -d alpine /bin/sh -c 'i=0; while true; do echo $i; i=$(expr $i + 1); sleep 1; done'
-
-sleep 1
-for i in $(seq 50); do
-	# docker start returns 0 silently if a container is already started
-	# docker checkpoint doesn't wait when docker updates a container state
-	# Due to both these points, we need to sleep after docker checkpoint to
-	# avoid races with docker start.
-	docker exec cr ps axf &&
+run_container () {
+	docker run --tmpfs /tmp --tmpfs /run --read-only --security-opt seccomp=unconfined --name cr -d alpine /bin/sh -c 'i=0; while true; do echo $i; i=$(expr $i + 1); sleep 1; done'
```

Comment:
The seccomp unconfined should not really be necessary anymore. I know it was needed for CentOS 7 because seccomp suspending was not backported, but besides that it should work